<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskAdd</class>
 <widget class="QDialog" name="taskAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item>
     <widget class="QTextEdit" name="descriptionTask"/>
    </item>
    <item>
     <widget class="QDateEdit" name="deadline">
      <property name="date">
       <date>
        <year>1999</year>
        <month>1</month>
        <day>17</day>
       </date>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>complete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
